File: 4기 코딩테스트_220615/4기 - 2nd 코딩테스트/CountEqualWithDividedNumber/CountEqualWithDividedNumber.py
def solution(nums, d):
    """
    :param nums: int[]
    :param d: int
    :return: int
    """

    result = 0

    for i in range(len(nums) - 1):
        for j in range(i + 1, len(nums)):
            if nums[i] == nums[j] and (nums[i] % d) == 0:
                result += 1

    return result

File: 4기 코딩테스트_220615/4기 - 2nd 코딩테스트/CountEqualWithDividedNumber/CountEqualWithDividedNumber.js
/**
 * @param nums {number[]}
 * @param d {number}
 * @return {number}
 */
function solution(nums, d) {
  let result = 0

  for (let i = 0; i < nums.length - 1; i++) {
    for (let j = i + 1; j < nums.length; j++) {
      if (nums[i] === nums[j] && (nums[i] % d) === 0) {
        result++
      }
    }
  }

  return result
}

export default solution
